### PR TITLE
Add French translation

### PR DESCRIPTION
**Pull request by huey735**

### CODE REVIEW

File: _data/fr.yml [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+  liReleaseNotes: Notes de publication
+  liVerification: Vérification
+  liVerificationText1: PGP signatures
+  liVerificationText2: Clé publique PGP
+
+dao:
+  hLearnMore: "Vous voulez en savoir plus?"
+  pLearnMore: "Vous pouvez en apprendre davantage sur la DAO de Bisq à partir des ressources ci-dessous, qui contiennent de la documentation complète de type ELI5, ainsi des vidéos avec les spécifications techniques pour les développeurs."
+  conceptualOverview: Documentation de vue d'ensemble du concept
+  pConceptualOverview: "<p class=\"bigger\">En langage simple<a href=\"https://docs.bisq.network/user-dao-intro.html\" target=\"_blank\" rel=\"noopener\">description</a> de la DAO.</p>"
```
Comment (m52go) [inline _data/fr.yml:93]:
```suggestion
  pConceptualOverview: "<p class=\"bigger\">En langage simple <a href=\"https://docs.bisq.network/user-dao-intro.html\" target=\"_blank\" rel=\"noopener\">description</a> de la DAO.</p>"
```

File: _data/fr.yml [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+  bisqDaoBrief: La DAO de Bisq en vidéos brèves
+  pBisqDaoBrief: "<p class=\"bigger\">Rapide, courte <a href=\"https://www.youtube.com/playlist?list=PLFH5SztL5cYPAXWFz-IMB4dBZ0MEZEG_e\" target=\"_blank\">série de vidéos</a> sur la DAO de Bisq.</p>"
+  technicalOverview: Vue d'ensemble technique - Documentation
+  pTechnicalOverview: "<p class=\"bigger\"><a href=\"https://docs.bisq.network/dao/specification.html\" target=\"_blank\" rel=\"noopener\">spécificité</a> du fonctionnement de la DAO de Bisq et du jeton BSQ.</p>"
```
Comment (m52go) [inline _data/fr.yml:97]:
```suggestion
  pTechnicalOverview: "<p class=\"bigger\"><a href=\"https://docs.bisq.network/dao/specification.html\" target=\"_blank\" rel=\"noopener\">Spécificité</a> du fonctionnement de la DAO de Bisq et du jeton BSQ.</p>"
```

File: _data/fr.yml [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+    <p class='stat-label'>Moyenne journalière des transactions</p> <p class='stat-figure'>94</p> <p class='stat-note'>En Juin 2019</p>
+  pSoftwareDownloads: >
+    <p class='stat-label'>Téléchargements du logiciel</p> <p class='stat-figure'>13,723</p> <p class='stat-note'>For v1.1.2 (approximation)</p>
+  pGithubContributors: >
+    <p class='stat-label'>Contributeurs GitHub </p> <p class='stat-figure'>172</p> <p class='stat-note'>En Juin 2019</p>
+  pTwitterFollowers: >
+    <p class='stat-label'>Followers sur Twitter </p> <p class='stat-figure'>17,588</p> <p class='stat-note'>En Juin 2019</p>
+  pSlackUsers: >
+    <p class='stat-label'> Utilisateurs Slack</p> <p class='stat-figure'>1,183</p> <p class='stat-note'>En Juin 2019</p>
+  pNetworkLoad: "<p class='monitor-note'>Pour connaître la sollicitation du réseau P2P, les métriques Tor et d'autres données réseau, veuillez visiter le site Web du <a href='https://monitor.bisq.network' target='_blank'>contrôleur de réseau Bisq </a>.</p>"
```
Comment (m52go) [inline _data/fr.yml:316]:
```suggestion
  pNetworkLoad: "<p class='monitor-note'>Pour connaître la sollicitation du réseau P2P, les métriques Tor et d'autres données réseau, veuillez visiter le site Web du <a href='https://monitor.bisq.network' target='_blank'>contrôleur de réseau Bisq</a>.</p>"
```

File: _data/fr.yml [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+      <li>Bisq ne détient pas de bitcoin. Tous sont conservés dans des adresses à signatures multiples plutôt que dans un portefeuille contrôlé par Bisq.</li>
+      <li>Bisq ne détient pas de devise nationale. La devise nationale est transférée directement d'un trader à un autre.</li>
+      <li>Bisq utilise un réseau pair-à-pair sur Tor. Cela signifie qu'il n'y a pas de serveurs à pirater ou à DDoS.</li>
+      <li>Bisq ne connaît pas l'identité des traders. Aucune donnée n'est stockée  au sujet de qui trade avec qui.</li>
+      <li>Bisq ne nécessite pas d'enregistrement. Cela signifie que la vie privée est préservée, qu'il n'y a pas de temps d'attente " d'approbation " et que le vol d'identité est impossible.</li>
+      <li>Bisq n'est pas une entreprise. Il s'agit d'un projet open-source organisé sous la forme d'une<a href="https://docs.bisq.network/user-dao-intro.html#what-is-a-dao" target="_blank" rel="noopener">Organisation autonome décentralisée (DAO)</a>.
```
Comment (m52go) [inline _data/fr.yml:147]:
```suggestion
      <li>Bisq n'est pas une entreprise. Il s'agit d'un projet open-source organisé sous la forme d'une <a href="https://docs.bisq.network/user-dao-intro.html#what-is-a-dao" target="_blank" rel="noopener">Organisation autonome décentralisée (DAO)</a>.
```

File: _data/fr.yml [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+  hTradeCost: "Combien cela coûte-t-il de trader sur Bisq?"
+  pTradeCost: >
+    <p>Pour trader sur Bisq, les utilisateurs paient (1) des frais de transaction à Bisq et (2) des frais de minage aux mineurs.</p> <strong>Frais de transaction</strong> <p>Les makers d'offres paient 0,1 % du montant de l'opération et les takers d'offres paient 0,3 % du montant de l'opération. Le minimum dans les deux cas est de 0,00005 BTC pour éviter les limites de dust.</p>
+
+    <strong>Frais de minage</strong> <p>Un maker qui propose une offre ne paie que des frais de minage associés aux frais de transaction de l'échange. Un taker d'offre paie des frais de minage associés aux frais de transaction de l'échange, et de la transaction de dépôt ainsi que de la transaction de paiement. C'est parce que les frais de minage fluctuent et qu'il est impossible pour celui qui sommet l'offre de savoir quels seront les frais de minage au moment où son offre sera acceptée.</p> <p>Bisq utilise<a href="https://bitcoinfees.earn.com/api/v1/fees/list" target="_blank">ce service</a> pour estimer les frais . Les utilisateurs peuvent voir  les frais de minage en vigueur après qu'ils aient créé ou accepté une offre.</p>
```
Comment (m52go) [inline _data/fr.yml:221]:
```suggestion
    <strong>Frais de minage</strong> <p>Un maker qui propose une offre ne paie que des frais de minage associés aux frais de transaction de l'échange. Un taker d'offre paie des frais de minage associés aux frais de transaction de l'échange, et de la transaction de dépôt ainsi que de la transaction de paiement. C'est parce que les frais de minage fluctuent et qu'il est impossible pour celui qui sommet l'offre de savoir quels seront les frais de minage au moment où son offre sera acceptée.</p> <p>Bisq utilise <a href="https://bitcoinfees.earn.com/api/v1/fees/list" target="_blank">ce service</a> pour estimer les frais. Les utilisateurs peuvent voir  les frais de minage en vigueur après qu'ils aient créé ou accepté une offre.</p>
```

File: _data/fr.yml [hunk 6]
```diff
@@ -0,0 +1,316 @@
...
...>Les arbitres sont choisis au hasard pour chaque transaction.</p> <p>Si le trader A ne confirme pas la réception d'un transfert en devises nationales dans le <a href='#trade-periods'>temps imparti</a>, un bouton pour contacter l'arbitre apparaîtra pour les deux traders. Le trader B pourra alors soumettre à l'arbitre la preuve qu'il a, dans les faits, envoyé la somme correspondante. Alternativement, si le trader B n'a jamais envoyé la somme requise, le trader A peut soumettre à l'arbitre la preuve que les fonds n'ont jamais été reçus.</p> <p>Les éléments de preuve soumis doivent être cryptographiquement sécurisés en utilisant<a href="https://tlsnotary.org/" target="_blank">PageSigner</a> ou des relevés bancaires signés numériquement. Si ces méthodes échouent, des voies plus traditionnelles peuvent être employées.</p> <p>D'après notre expérience, la grande majorité des litiges ne sont pas causés par des intentions malveillantes, mais par des problèmes mineurs tels que des problèmes de réseau, des bugs, des problèmes bancaires, d'utilisation, etc.</p> <p>Une fois que l'arbitre aura clarifié la situation, il signera l'adresse multisignature 2-sur-3 en faveur de la partie concernée.</p>
```
Comment (m52go) [inline _data/fr.yml:233]:
```suggestion
    <p>Sur Bisq, les arbitres déterminent les solutions aux litiges lors des échanges. Les arbitres doivent payer un montant très élevé en <a href="https://docs.bisq.network/dao/phase-zero.html" target="_blank" rel="noopener">BSQ bond</a>, ce qui garantit des évaluations équitables et prévient la collusion avec les traders.</p> <p>Les arbitres sont choisis au hasard pour chaque transaction.</p> <p>Si le trader A ne confirme pas la réception d'un transfert en devises nationales dans le <a href='#trade-periods'>temps imparti</a>, un bouton pour contacter l'arbitre apparaîtra pour les deux traders. Le trader B pourra alors soumettre à l'arbitre la preuve qu'il a, dans les faits, envoyé la somme correspondante. Alternativement, si le trader B n'a jamais envoyé la somme requise, le trader A peut soumettre à l'arbitre la preuve que les fonds n'ont jamais été reçus.</p> <p>Les éléments de preuve soumis doivent être cryptographiquement sécurisés en utilisant <a href="https://tlsnotary.org/" target="_blank">PageSigner</a> ou des relevés bancaires signés numériquement. Si ces méthodes échouent, des voies plus traditionnelles peuvent être employées.</p> <p>D'après notre expérience, la grande majorité des litiges ne sont pas causés par des intentions malveillantes, mais par des problèmes mineurs tels que des problèmes de réseau, des bugs, des problèmes bancaires, d'utilisation, etc.</p> <p>Une fois que l'arbitre aura clarifié la situation, il signera l'adresse multisignature 2-sur-3 en faveur de la partie concernée.</p>
```

File: _data/fr.yml [hunk 7]
```diff
@@ -0,0 +1,316 @@
...
+  hCollusion: "Comment prévenir la collusion entre les arbitres et les traders?"
+  pCollusion: >
+    <p>Comme mentionné plus haut, les arbitres doivent bloquer un <a href="https://docs.bisq.network/dao/phase-zero.html" target="_blank" rel="noopener">BSQ bond</a>élevé pour devenir arbitre en premier lieu - beaucoup plus élevé que la limite de transaction. Si les deux traders ne sont pas d'accord avec la résolution d'un arbitre, ils peuvent demander la confiscation du BSQ bond de l'arbitre.</p> <p>S'il est établi qu'il y a eu collusion, l'arbitre qui s'est mal conduit perdra son cautionnement en BSQ bond. L'arbitre subit donc une perte importante, car le BSQ bond était beaucoup plus élevé que le montant obtenu en étant malhonnête.</p>
```
Comment (m52go) [inline _data/fr.yml:236]:
```suggestion
    <p>Comme mentionné plus haut, les arbitres doivent bloquer un <a href="https://docs.bisq.network/dao/phase-zero.html" target="_blank" rel="noopener">BSQ bond</a> élevé pour devenir arbitre en premier lieu - beaucoup plus élevé que la limite de transaction. Si les deux traders ne sont pas d'accord avec la résolution d'un arbitre, ils peuvent demander la confiscation du BSQ bond de l'arbitre.</p> <p>S'il est établi qu'il y a eu collusion, l'arbitre qui s'est mal conduit perdra son cautionnement en BSQ bond. L'arbitre subit donc une perte importante, car le BSQ bond était beaucoup plus élevé que le montant obtenu en étant malhonnête.</p>
```

File: _data/fr.yml [hunk 8]
```diff
@@ -0,0 +1,316 @@
...
+    <p>Jusqu'à présent, Bisq a intentionnellement évité d'offrir un moyen d'entrer en contact direct avec son pair de trading Il n'y a généralement pas besoin d'un tel mécanisme, et offrir un tel mécanisme n'ouvrirait qu'une " boîte de Pandore " en matière de sécurité : les utilisateurs malveillants pourraient piéger leurs pairs par le bias d'un échange en direct sans utiliser Bisq, puis exploiter le manque de sécurité pour arnaquer l'autre personne, ou utiliser le canal direct pour d'autres activités frauduleuses (fraudes d'ingénierie sociale, spam, etc).</p> <p>Au lieu de cela, en cas de problèmes, les utilisateurs peuvent ouvrir un différend et le résoudre avec l'aide d'un arbitre. Il y a également le <a href='https://bisq.community' target='_blank'>forum de Bisq</a> et<a href='https://bisq.network/slack-invite' target='_blank'>le canal de discussion Slack</a> à la disposition des utilisateurs désireux d'obtenir de l'aide.</p> <p>Remarque : avec le lancement du <a href='https://github.com/bisq-network/proposals/issues/52' target='_blank'>nouveau protocole d' échange</a>, il y aura, cependant, un moyen pour les pairs de communiquer directement les uns avec les autres.</p>
```
Comment (m52go) [inline _data/fr.yml:251]:
```suggestion
    <p>Jusqu'à présent, Bisq a intentionnellement évité d'offrir un moyen d'entrer en contact direct avec son pair de trading Il n'y a généralement pas besoin d'un tel mécanisme, et offrir un tel mécanisme n'ouvrirait qu'une " boîte de Pandore " en matière de sécurité : les utilisateurs malveillants pourraient piéger leurs pairs par le bias d'un échange en direct sans utiliser Bisq, puis exploiter le manque de sécurité pour arnaquer l'autre personne, ou utiliser le canal direct pour d'autres activités frauduleuses (fraudes d'ingénierie sociale, spam, etc).</p> <p>Au lieu de cela, en cas de problèmes, les utilisateurs peuvent ouvrir un différend et le résoudre avec l'aide d'un arbitre. Il y a également le <a href='https://bisq.community' target='_blank'>forum de Bisq</a> et <a href='https://bisq.network/slack-invite' target='_blank'>le canal de discussion Slack</a> à la disposition des utilisateurs désireux d'obtenir de l'aide.</p> <p>Remarque : avec le lancement du <a href='https://github.com/bisq-network/proposals/issues/52' target='_blank'>nouveau protocole d' échange</a>, il y aura, cependant, un moyen pour les pairs de communiquer directement les uns avec les autres.</p>
```

File: _data/fr.yml [hunk 9]
```diff
@@ -0,0 +1,316 @@
...
+  pBisqDaoWhat: >
+    <p>La DAO de Bisq (organisation autonome décentralisée) est un système de gouvernance décentralisé pour le logiciel Bisq. Il prend la place d'une entreprise. Comment un logiciel décentralisé peut-il être efficace s'il est toujours contrôlé par une seule entité, comme une entreprise?</p> <p>Un logiciel décentralisé ne sert à rien sans une gouvernance décentralisée.</p> <p>C'est pourquoi Bisq n'est pas une société ou une entité juridique de quelque nature que ce soit, elle est plutôt organisée comme un DAO. La DAO s'occupe du financement et de l'élaboration de la stratégie du logiciel afin de permettre au réseau Bisq de prospérer et, en même temps, de le renforcer contre les attaques contre ses infrastructures qui sont le moteur de sa direction et son fonctionnement.</p> <p>En résumé, la DAO de Bisq permet à Bisq de renforcer encore sa résistance à la censure, un <a href='https://docs.bisq.network/intro.html' target='_blank'>principe fondateur du projet</a> depuis sa genèse.</p> <p>En savoir plus sur la DAO de Bisq Dans<a href='https://docs.bisq.network/user-dao-intro.html' target='_blank'>ce document de présentation</a>.</p>
```
Comment (m52go) [inline _data/fr.yml:255]:
```suggestion
    <p>La DAO de Bisq (organisation autonome décentralisée) est un système de gouvernance décentralisé pour le logiciel Bisq. Il prend la place d'une entreprise. Comment un logiciel décentralisé peut-il être efficace s'il est toujours contrôlé par une seule entité, comme une entreprise?</p> <p>Un logiciel décentralisé ne sert à rien sans une gouvernance décentralisée.</p> <p>C'est pourquoi Bisq n'est pas une société ou une entité juridique de quelque nature que ce soit, elle est plutôt organisée comme un DAO. La DAO s'occupe du financement et de l'élaboration de la stratégie du logiciel afin de permettre au réseau Bisq de prospérer et, en même temps, de le renforcer contre les attaques contre ses infrastructures qui sont le moteur de sa direction et son fonctionnement.</p> <p>En résumé, la DAO de Bisq permet à Bisq de renforcer encore sa résistance à la censure, un <a href='https://docs.bisq.network/intro.html' target='_blank'>principe fondateur du projet</a> depuis sa genèse.</p> <p>En savoir plus sur la DAO de Bisq dans <a href='https://docs.bisq.network/user-dao-intro.html' target='_blank'>ce document de présentation</a>.</p>
```

File: _data/fr.yml [hunk 10]
```diff
@@ -0,0 +1,316 @@
...
+  pMultisig: >
+    <p>“<a href="https://en.bitcoin.it/wiki/Multisignature" target="_blank">Multisig</a>”, comme on l'appelle, est intégré à Bitcoin comme une simple version d'un contrat intelligent. Une adresse multisig est générée avec plusieurs clés publiques avec une spécification du nombre de clés à signer pour libérer les fonds.</p> <p>Bisq utilise des adresses multisig2-sur-3. Cela signifie que trois clés publiques sont utilisées pour créer l'adresse de paiement (celle de l'acheteur, du vendeur et d'un arbitre) et que la transaction doit être signée par deux des trois parties pour que les fonds soient libérés.</p> <p>En pratique, l'acheteur et le vendeur signent chacun la transaction pour débloquer les fonds. Mais en cas de litige, un arbitre se rangera du côté de l'une des parties pour fournir la deuxième signature nécessaire à la libération des fonds.</p>
+  hOtherQuestions: "J'ai une question qui n'est pas abordée ici...?"
+  pOtherQuestions: "<p>Veuillez poster votre question sur <a href=\"https://bisq.community/\" target=\"_blank\"> le Forum de Bisq</a> ou<a href='https://bisq.network/slack-invite' target='_blank'>sur le canal de discussion Slack</a>.</p>"
```
Comment (m52go) [inline _data/fr.yml:292]:
```suggestion
  pOtherQuestions: "<p>Veuillez poster votre question sur <a href=\"https://bisq.community/\" target=\"_blank\"> le Forum de Bisq</a> ou <a href='https://bisq.network/slack-invite' target='_blank'>sur le canal de discussion Slack</a>.</p>"
```

File: _data/fr.yml [hunk 11]
```diff
@@ -0,0 +1,316 @@
...
+    </p> <p>Si le bouton n'apparaît pas, vous pouvez ouvrir un ticket de support en appuyant sur Cmd+o (ou Ctrl+o, selon votre système) lorsque la transaction est sélectionnée dans Portfolio > Open Trades list.</p>
+  hMultisg: "Qu'est-ce qu'une ‘adresse multisignature’ Bitcoin?"
+  pMultisig: >
+    <p>“<a href="https://en.bitcoin.it/wiki/Multisignature" target="_blank">Multisig</a>”, comme on l'appelle, est intégré à Bitcoin comme une simple version d'un contrat intelligent. Une adresse multisig est générée avec plusieurs clés publiques avec une spécification du nombre de clés à signer pour libérer les fonds.</p> <p>Bisq utilise des adresses multisig2-sur-3. Cela signifie que trois clés publiques sont utilisées pour créer l'adresse de paiement (celle de l'acheteur, du vendeur et d'un arbitre) et que la transaction doit être signée par deux des trois parties pour que les fonds soient libérés.</p> <p>En pratique, l'acheteur et le vendeur signent chacun la transaction pour débloquer les fonds. Mais en cas de litige, un arbitre se rangera du côté de l'une des parties pour fournir la deuxième signature nécessaire à la libération des fonds.</p>
```
Comment (m52go) [inline _data/fr.yml:290]:
```suggestion
    <p>“<a href="https://en.bitcoin.it/wiki/Multisignature" target="_blank">Multisig</a>”, comme on l'appelle, est intégré à Bitcoin comme une simple version d'un contrat intelligent. Une adresse multisig est générée avec plusieurs clés publiques avec une spécification du nombre de clés à signer pour libérer les fonds.</p> <p>Bisq utilise des adresses multisig 2-sur-3. Cela signifie que trois clés publiques sont utilisées pour créer l'adresse de paiement (celle de l'acheteur, du vendeur et d'un arbitre) et que la transaction doit être signée par deux des trois parties pour que les fonds soient libérés.</p> <p>En pratique, l'acheteur et le vendeur signent chacun la transaction pour débloquer les fonds. Mais en cas de litige, un arbitre se rangera du côté de l'une des parties pour fournir la deuxième signature nécessaire à la libération des fonds.</p>
```

File: _data/fr.yml [hunk 12]
```diff
@@ -0,0 +1,316 @@
...
+
+    <p>Nous sommes conscients que cette exigence peut être un obstacle pour les débutants, mais il existe de nombreuses façons d'obtenir votre premier bitcoin : amis et famille, rencontres Bitcoin, guichets automatiques Bitcoin, bons d'achat, contribuer, etc.</p>
+  hPaymentSupport: "Est-ce que Bisq prend en charge les paiements ‘XYZ’?"
+  pPaymentSupport: >
+    <p>Bisq propose déjà un large éventail de modes de paiement pour les devises nationales, mais de nouveaux moyens de paiement pourront être ajoutés aussi longtemps que:</p> <ul>
+      <li>les rétrofacturations sont peu probables</li>
+      <li>les arbitres peuvent consulter la preuve de la transaction</li>
+    </ul> <p>Les virements bancaires nationaux sont pris en charge dans tous les pays.</p> <p>Yvous pouvez proposer un nouveau moyen de paiement sur le forum</a>, <a href='https://bisq.network/slack-invite' target='_blank'>Slack</a>, ou<a href='https://github.com/bisq-network/bisq/issues' target='_blank'>GitHub</a>.</p>
```
Comment (m52go) [inline _data/fr.yml:282]:
```suggestion
    </ul> <p>Les virements bancaires nationaux sont pris en charge dans tous les pays.</p> <p>Yvous pouvez proposer un nouveau moyen de paiement sur le forum</a>, <a href='https://bisq.network/slack-invite' target='_blank'>Slack</a>, ou <a href='https://github.com/bisq-network/bisq/issues' target='_blank'>GitHub</a>.</p>
```

File: _data/fr.yml [hunk 13]
```diff
@@ -0,0 +1,316 @@
...
+  technicalOverview: Vue d'ensemble technique - Documentation
+  pTechnicalOverview: "<p class=\"bigger\"><a href=\"https://docs.bisq.network/dao/specification.html\" target=\"_blank\" rel=\"noopener\">spécificité</a> du fonctionnement de la DAO de Bisq et du jeton BSQ.</p>"
+  notConvinced: "Vous n'êtes pas convaincu qu'un jeton est nécessaire?"
+  pNotConvinced: "<p class=\"bigger\">Nous avons un <a href=\"https://bisq.network/blog/bisq-dao-for-bitcoin-maximalists/\" target=\"_blank\">message</a> rien que pour vous.</p>"
+  seeMore: >
+    <p>Consultez d'autres ressources sur notre <a href='https://docs.bisq.network/dao.html'>Tout savoir sur la DAO de Bisq</a> doc.</p> <p>Anciens contributeurs, vérifiez <a href='https://docs.bisq.network/compensation.html#get-added-to-genesis'>ce lien</a> pour obtenir des instructions sur la façon d'être inclus dans la distribution genesis de BSQ.</p>
+downloads:
+  hBisqDownloads: Téléchargements Bisq
+  runningWindows: Vous semblez utiliser Windows.
```
Comment (m52go) [inline _data/fr.yml:104]:
```suggestion
  runningWindows: Vous semblez utiliser Windows
```

File: _data/fr.yml [hunk 14]
```diff
@@ -0,0 +1,316 @@
...
+  runningMac: Vous semblez utiliser un Mac.
```
Comment (m52go) [inline _data/fr.yml:105]:
```suggestion
  runningMac: Vous semblez utiliser un Mac
```

File: _data/fr.yml [hunk 15]
```diff
@@ -0,0 +1,316 @@
...
+  runnigDebian: Vous semblez utiliser Debian/Ubuntu.
```
Comment (m52go) [inline _data/fr.yml:106]:
```suggestion
  runnigDebian: Vous semblez utiliser Debian/Ubuntu
```

File: _data/fr.yml [hunk 16]
```diff
@@ -0,0 +1,316 @@
...
+  runningRedHat: Vous semblez utiliser Debian/Ubuntu.
```
Comment (m52go) [inline _data/fr.yml:107]:
```suggestion
  runningRedHat: Vous semblez utiliser Debian/Ubuntu
```

File: _data/fr.yml [hunk 17]
```diff
@@ -0,0 +1,316 @@
...
+  liGetStarted3: >
+    <li class="mt-0 mb-4 pb-4 border-bottom step step-text-2" data-image="step-image-2">Ajoutez vos comptes en devises nationales ou cryptos.</li> <li class="mt-0 mb-4 pb-4 border-bottom step step-text-3" data-image="step-image-3">Créez une nouvelle offre ou choisissez une offre existante.</li> <li class="mt-0 step step-text-4" data-image="step-image-4">Suivez l'interface utilisateur pour finaliser la transaction.</li>
+  hJoin: Rejoignez la communauté
+  pJoin: >
+    <p>Bisq est toujours à la recherche de développeurs, de concepteurs et d'experts en sécurité hautement qualifiés pour se joindre à notre <a href="https://bisq.network/community/">communauté</a>. Si vous souhaitez contribuer à l'amélioration de Bisq, il existe de nombreuses façons de <a href="https://docs.bisq.network/contributor-checklist.html" target="_blank" rel="noopener">contribuer</a>. Nous aimerions beaucoup <a href="https://bisq.network/community/">entendre parler de vous</a>!</p> <p>Notre<a href="https://bisq.network/roadmap/">feuille de route</a> fournit une vue d'ensemble des fonctionnalités complexes et des étapes importantes sur le chemin vers Bisq v1.0.</p>
```
Comment (m52go) [inline _data/fr.yml:56]:
```suggestion
    <p>Bisq est toujours à la recherche de développeurs, de concepteurs et d'experts en sécurité hautement qualifiés pour se joindre à notre <a href="https://bisq.network/community/">communauté</a>. Si vous souhaitez contribuer à l'amélioration de Bisq, il existe de nombreuses façons de <a href="https://docs.bisq.network/contributor-checklist.html" target="_blank" rel="noopener">contribuer</a>. Nous aimerions beaucoup <a href="https://bisq.network/community/">entendre parler de vous</a>!</p> <p>Notre <a href="https://bisq.network/roadmap/">feuille de route</a> fournit une vue d'ensemble des fonctionnalités complexes et des étapes importantes sur le chemin vers Bisq v1.0.</p>
```

File: _includes/main_nav_tr.html [hunk 18]
```diff
@@ -1,6 +1,13 @@
  <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top pl-md-5 pr-md-5 pl-sm-3 pr-0">
 
-    <a class="navbar-brand" href="{{ site.url }}/{{ page.lang }}/"><img src="{{ site.url }}/images/bisq-logo.svg" height="30"/></a>
+   <div>
+     <a class="navbar-brand" href="{{ site.url }}/{{ page.lang }}/"><img src="{{ site.url }}/images/bisq-logo.svg" height="30"/></a>
+
+     <!-- Outdated translation alert tool tag-->
```
Comment (m52go) [inline _includes/main_nav_tr.html:6]: Please remove the commit with this change, since there's already another PR with this change.

**Review by m52go — CHANGES_REQUESTED** [review]
Lots of minor issues (mainly spacing), which I made suggestions to. Otherwise it makes the project look sloppy. In the future, please run the site locally and do a quick glance through the pages to make sure there aren't any lingering gremlins.

Also you have one commit related to outdated_translations here...please remove it (commit 20a08bcdb9ce2965c61b3c34681cf7c024cca2d0) since it's not related to this PR.

**Comment by huey735** [comment]
Closing because it contains unrelated commit.

**Comment by m52go** [comment]
@huey735 you should be able to remove the commit and then force-push (`git push -f <remote> addFrenchTranslation`). That would allow you to easily implement all my suggested edits, instead of doing them over on your own and me reviewing again.
